spectator-api/src/main/java/com/netflix/spectator/api/BasicTag.java

```diff
@@ -66,16 +65,14 @@ public String value() {
     return key.equals(other.key) && value.equals(other.value);
   }
 
-  /**
-   * This object is immutable and the hash code is precomputed in the constructor. The id object
-   * is typically created to lookup a Meter based on dynamic dimensions so we assume that it is
```

Comment:
Maybe this used to be true, but DefaultId stores its tags in an `ArrayTagSet`, so the hashCode method doesn't come into the picture.